<launch>
    <include file="$(find mission_commander)/launch/mission_commander_sim.launch"/>
    <include file="$(find drone_sim)/launch/apm.launch" />
    <node pkg="drone_ridder" type="drone_ridder" name="drone_ridder" output="screen" />
    <include file="$(find object_global_localizator)/launch/object_global_localizator_sim.launch"/>
    <node pkg="trajectory_planer" type="trajectory_planer" name="trajectory_planer" output="screen" />
    <!--><-->
    <include file="$(find darknet_ros)/launch/darknet_ros.launch" />

</launch>